dags/topline.py

```diff
@@ -0,0 +1,46 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'amiyaguchi@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2017, 4, 23),
```

Comment:
I set it to 2017-03-26, which is the last Sunday in March. This should give a single monthly run and a few weekly runs.